cdap-api/src/main/java/co/cask/cdap/api/flow/flowlet/AbstractFlowlet.java

```diff
@@ -130,31 +120,6 @@ protected void useDatasets(Iterable<String> datasets) {
     configurer.useDatasets(datasets);
   }
 
-  /**
-   * Default constructor that uses {@link #getClass()}.{@link Class#getSimpleName() getSimpleName} as the
-   * flowlet name.
-   */
-  protected AbstractFlowlet() {
```

Comment:
what happens if the user doesn't call setName()? Does the configurer default it to the class simple name?